manifests/charts/istio-cni/values.yaml

```diff
@@ -8,7 +8,7 @@ defaults:
 
     # Configuration log level of istio-cni binary
```

Comment:
I would word this as

```
Log level of the `istio-cni` plugin binary. This will impact the logs written to kubelet, and to the CNI DaemonSet.
Note: the visibility of the logs within the CNI DaemonSet is additionally controlled by `global.logging.level` under the `cni-plugin` scope. For instance, use `global.logging.level=cni-plugin:debug` to enable debug level logs.
```

Note: cni-agent -> cni-plugin is due to https://github.com/istio/istio/commit/942eee157989a5e6b6c781d6f4af979935d9facb

Reply:
(I don't really care too much on the wording beyond the agent->plugin part)

Reply:
This doesn't just control the plugin log level tho. It controls the `istio-cni` log level - which includes both the plugin and the agent, as they work in concert.

You would control plugin logs with scoping, e.g.

`logLevel: cni-plugin:info,cni-agent:debug`
 
 but we probably don't need to mention `plugin` here, and get into distinctions between the plugin and the agent, which someone reading this is not likely to know or care about.
 
 If we do, let's at least suggest `all:debug`, and then "if you want to constrain plugin logs separately from the agent logs, use `cni-plugin:<whatever>,cni-agent:<whatever-else>`"?